src/main/java/uk/gov/beis/els/categories/airconditioners/model/MultipleClimateGroupForm.java

```diff
@@ -15,18 +17,25 @@
 
   @FieldPrompt("Energy efficiency class for Warmer heating season")
   @NotBlank(groups = WarmerClimateGroup.class, message = "Select an energy efficiency indicator for warmer climate conditions")
+  @ApiValuesFromLegislationCategory(serviceClass = AirConditionersService.class)
   private String warmerHeatingEfficiencyRating;
 
   @FieldPrompt("Design load for heating in warmer climate conditions in kW")
+  @NotNull(groups = WarmerClimateGroup.class)
   @Digits(groups = WarmerClimateGroup.class, integer = 2, fraction = 1, message = "Enter the design load for warmer climate conditions, up to 2 digits with an optional decimal place")
+  @Schema(type = "number")
   private String warmerHeatingDesignLoad;
 
   @FieldPrompt("Seasonal coefficient of performance in warmer climate conditions (SCOP value)")
+  @NotNull(groups = WarmerClimateGroup.class)
```

Comment:
do we want to show this as mandatory in the doco even if they are not required if the user selects `warmerConditions` as `false`?